docs/setup/settings.asciidoc

```diff
@@ -39,6 +39,12 @@ Elasticsearch. This value must be a positive integer.
 
 `elasticsearch.shardTimeout:`:: *Default: 30000* Time in milliseconds for Elasticsearch to wait for responses from shards. Set to 0 to disable.
 
+`elasticsearch.sniffInterval:`:: *Default: false* Time in milliseconds between requests to check Elasticsearch for an updated list of nodes.
+
+`elasticsearch.sniffOnStart:`:: *Default: false* Attempt to find other Elasticsearch nodes on startup.
+
+`elasticsearch.sniffOnConectionFault:`:: *Default: false* Update the list of Elasticsearch nodes immediately following a connection fault.
```

Comment:
Hello, looks like some bits fell out. Connection is missing an 'n' in the setting name.